src/admin-views/widgets/components/radio.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Admin View: Widget Radio Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/radio.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string       $label   Label for the radio group.
+ * @var string       $value   Value for the radio.
+ * @var string       $name    Name attribute for the radio.
+ * @var array<array> $options Data for the individual radio buttons.
+ */
+if ( ! empty( $label ) ) : ?>
+	<p><?php echo esc_html( $label ); ?></p>
+<?php
+endif;
+
+$i = 1;
+
+foreach ( $options as $option ) :
+    $radio_id = $id . $i;
+    ?>
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: c6a29a3221bc1b78a77a43bde377f3c8</sub>